content/docs/contributing/docs.md

```diff
@@ -57,7 +57,7 @@ the website engine itself. Source code and content files need to be properly
 formatted and linted as well, which is also ensured by the full setup below.
 
 Make sure you have [Python](https://www.python.org/downloads/) 3.8+, a recent
```

Comment:
Also not sure this is true. Why is Python needed in the first place again? I don't have `puthon` command in my system and this works. Also, my system default (`python3`) is 3.1 and again, this works fine. Cc @rogermparent thanks

Reply:
I think we could drop this, it may have been a legacy requirement or something because I also don't remember any Python dependency in our websites barring the DVC images on iterative.ai, which doesn't apply here.

Reply:
```suggestion
Make sure you have a recent
```